etc/docker/Dockerfile

```diff
@@ -99,6 +99,7 @@ RUN rm -rf $CARGO_HOME/registry
 RUN echo "PATH=$PATH" >> ~/.bash_profile && \
     echo ". $ESP_ROOT/export-esp.sh" >> ~/.bash_profile && \
     echo ". $IDF_PATH/export.sh"  >> ~/.bash_profile && \
+    echo "ESP_IDF_TOOLS_INSTALL_DIR=fromenv"  >> ~/.bash_profile && \
```

Comment:
This is the change we need to get into the canon image before we can release the rest of the change. The effect of this is to override
```
[package.metadata.esp-idf-sys]
esp_idf_version = "v4.4.4"
```
In the relevant `Cargo.toml`